var/spack/repos/builtin/packages/charm/package.py

```diff
@@ -47,63 +50,123 @@ class Charm(Package):
 
     # Support OpenMPI; see
     # <https://charm.cs.illinois.edu/redmine/issues/1206>
-    patch("mpi.patch")
+    # Patch is no longer needed in versions 6.8.0+
+    patch("mpi.patch", when="@:6.7.1")
+
     # Ignore compiler warnings while configuring
     patch("strictpass.patch")
 
+    # Build targets
+    # "target" is reserved, so we have to use something else.
+    variant(
```

Comment:
is it multivalued?

Reply:
yes

Reply:
Can more than one value be requested at the same time? If so, it needs `multi=True`.